routes/api/users/index.js

```diff
@@ -60,6 +60,12 @@ router.post('/:user_id/status', authorization.needed('ADMIN'), (req, res, next)
     .catch(next);
 });
 
+router.post('/:user_id/username-enable', authorization.needed('ADMIN'), (req, res, next) => {
+  UsersService
+    .toggleNameEdit(req.params.user_id, true)
+    .then(() => {
+      res.status(204).end();
+
 router.post('/:user_id/displayname', authorization.needed(), (req, res, next) => {
```

Comment:
Going forward we should not add any more REST routes unless they pertain to auth.